samples/gradle.properties

```diff
@@ -0,0 +1,7 @@
+android.useAndroidX=true
+# Mandatory properties in root project for insturmentation plugin TODO remove all of them
+teamcityBuildId=1
```

Comment:
it strange that we need this garbage there

Reply:
I'm about teamcityBuildId teamcityUrl buildNumber teamcityBuildType

Reply:
it should be removed yes, feel free to do it